src/App.js

```diff
@@ -1,13 +1,111 @@
-import React from 'react';
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import "./components/TodoComponents/Todo.css";
+
+const initialTodos = [
+  { id: "1559140530360", task: "Pay the electricity bill", complete: false },
+  {
+    id: "1559140546041",
+    task: "Book an appointment at the embassy",
+    complete: true
+  },
+  {
+    id: "1559140554289",
+    task: "Learn how to make a croissant",
+    complete: false
+  }
+];
 
 class App extends React.Component {
```

Comment:
I would try to keep the App.js mostly presentational and add an other container component holding the logic (methods & state). This will help you if you want to add features to the app afterwards